go.mod

```diff
@@ -30,7 +30,7 @@ require (
 	github.com/jrick/logrotate v1.0.0
 	github.com/jrick/wsrpc/v2 v2.3.1
 	go.etcd.io/bbolt v1.3.3
-	golang.org/x/crypto v0.0.0-20190820162420-60c769a6c586
+	golang.org/x/crypto v0.0.0-20200214034016-1d94cc7ab1c6
```

Comment:
x/crypto was bumped here because the following commit I was working on imported chacha20, which was not in this older version.